### ISSUE TITLE
[Feature] Re-implement the old main screen aesthetic

### ISSUE THREAD

**Issue by Palloxin** [AniLibApp/AniLib#99]
## Description
The old main screen UX is better. Nothing much to say. Newer doesnt mean better(aesthetically in this case)
2.0.0:
![image](https://github.com/user-attachments/assets/0e9e4cc1-e5a7-485a-99c7-882f2a0f1551)

1.1.19:
![image](https://github.com/user-attachments/assets/1363336a-5047-44a4-ac32-a6922ab38bfe)

==
These.
2.0.0:
![Screenshot_20241014-212611_1](https://github.com/user-attachments/assets/8ee0b530-dfac-477d-badf-e0ef0ef41c9c)

1.1.19
![Screenshot_20241014-212749_1](https://github.com/user-attachments/assets/e1d49995-a56f-4ed1-8ae8-e52f960ab879)

==
This as a whole was better:
1.1.19
![Screenshot_2024-10-14-21-28-50_1](https://github.com/user-attachments/assets/6a246258-5f32-4d6a-a571-22ba11e47617)

2.0.0
![Screenshot_2024-10-14-21-31-51_1](https://github.com/user-attachments/assets/4d630e8d-1c0a-4ae7-bc9e-01189f7114fc)

==
 2.0.0
![Screenshot_20241014-213828_1](https://github.com/user-attachments/assets/e2b40bea-a419-4b55-957a-169657e85006)

1.1.19
![Screenshot_20241014-213338_1](https://github.com/user-attachments/assets/98bfc4a2-f2dc-4fa2-9c52-c323a337c499)

==

2.0.0
![Screenshot_20241014-213125_1](https://github.com/user-attachments/assets/a71dcb35-eb0d-49b0-bb31-3b64ce1ee3d4)

1.1.19
![Screenshot_20241014-213328_1](https://github.com/user-attachments/assets/6fb882a9-0db9-4e11-b6b9-d97decda2d1a)

## Other details
Of course, not reverse it, but re-create it.

**Comment by Palloxin** [comment]
PR, the issues template need a separation for Bugs and Feature requests.

**Comment by Palloxin** [comment]
There is also more spacing between the sections, like between Airing and Popular

**Comment by Moonchildish** [comment]
I agree, I liked the aesthetics and ui of the older version better. Since the new version is started having several bugs too..
